entry.js

```diff
@@ -64,6 +71,8 @@ export function setTodos(newTodos) {
   todos = newTodos;
 }
 
+searchTodoInput.addEventListener("input", debounceSearch(searchTodo, 500));
```

Comment:
If for some reason we want to pass some arguments to the event listener(normally browser sends the `event` object to the listener), we wouldn't be able to access the arguments or the events. Can you figure out why and the solution to that?